From 857ba2906a2bb2d7160f74b8df045185fe215d82 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sun, 11 Jun 2017 08:05:04 +0200
Subject: [PATCH] vcarddir: add cmake dependency on akonadi-singlefileresource

the vcarddir resources uses the generated source from settingsdialog.ui
of singlefileresource; make sure it is already generated
---
 resources/vcarddir/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/resources/vcarddir/CMakeLists.txt b/resources/vcarddir/CMakeLists.txt
index 28fdd048f..c2f58aec6 100644
--- resources/vcarddir/CMakeLists.txt
+++ resources/vcarddir/CMakeLists.txt
@@ -36,6 +36,8 @@ target_link_libraries(akonadi_vcarddir_resource
   Qt5::DBus
 )
 
+add_dependencies(akonadi_vcarddir_resource akonadi-singlefileresource)
+
 add_subdirectory(wizard)
 
 install(TARGETS akonadi_vcarddir_resource ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
